docs/develop/module-specifications/spec-governance.mdx

```diff
@@ -35,31 +44,51 @@ Deposits are burned under any of the following conditions:
 - `Quorum` is not met: the number of total votes after the one-week voting period is less than 10% of all staked Luna.
 - The number of `NoWithVeto` votes is above 33.4% of the total vote.
 
-### Voting Period
+### Voting period
+
+If the minimum deposit has been reached before the end of the deposit period, then the proposal goes into a one-week voting period. While the proposal is in voting, holders of staked Luna can cast votes for the proposal. 
+
+### Voting options 
 
-If the minimum deposit has been reached before the end of the deposit period, then the proposal goes into a one-week voting period. While the proposal is in voting, holders of staked Luna can cast votes for the proposal. The 4 voting options available are:
+The 4 voting options available are:
 
-- `Yes` - in favor
-- `No` - not in favor
-- `NoWithVeto` - veto
-- `Abstain` - does not influence vote
+- `Yes`: in favor.
+- `No`: not in favor.
+- `NoWithVeto`: not in favor, and the creator should lose their deposit. 
```

Comment:
I think the comma may not be needed in this sentence.

Reply:
"the creator should lose their deposit" is an independent clause, so a comma is needed.